src/NzbDrone.Common/Http/HttpRequest.cs

```diff
@@ -54,6 +54,7 @@ public Uri Url
         public Dictionary<string, string> Cookies { get; private set; }
         public bool StoreResponseCookie { get; set; }
         public TimeSpan RateLimit { get; set; }
+        public HttpRequestProxySettings Proxy {get; set;}
```

Comment:
That sounds like a good idea to me. For my benefit, the reason to do this is so it doesn't need to be set all over the place to be used?